app/views/user_mailer/admin_hidden_work_notification.html.erb

```diff
@@ -1,12 +1,12 @@
 <% content_for :message do %>
-    <p>Dear <%= @user.default_pseud.byline %>,</p>
+<% @pretty_title = "<i>#{style_bold(@work.title.html_safe)}</i>".html_safe %>
```

Comment:
The work title is a link in the current version, so I think this should be more like: 
```
<% pretty_title = "<i>#{style_link(@work.title.html_safe, work_url(@work))}</i>".html_safe %>
```

Also,  just plain `pretty_title` should work fine. I think the reason it complained before was a mistake in the en.yml part, which I will comment on down there.